forte/processors/base/data_augment_processor.py

```diff
@@ -94,23 +618,40 @@ def default_configs(cls):
             - augment_entries: defines the entries the processor
             will augment. It should be a full path to the entry class.
             - other_entry_policy: a dict specifying the policies for
-            other entries.
+            other entries. It has two keys: "entry" and "policy".
 
-            If "auto_align", the span of the entry will be automatically
-            modified according to its original location. However, some
-            spans might become invalid after the augmentation, for
-            example, the tokens within a replaced sentence may disappear.
+            The values of "policy" are aligned with the values of "entry".
+            They are two aligned lists. The "entry" should be class name.
+            The "policy" specifies how to process the corresponding entries
+            after replacement.
 
-            Entries not in the dict will not be copied to the new data pack.
+            If the policy is "auto_align", the span of the entry
```

Comment:
The documentation here only talks about the treatment to `Annotation`, so don't use the word `entry`. And what happens to other types of entry such as Link and Group?